mirrord/cli/src/container/command_builder.rs

```diff
@@ -57,19 +61,28 @@ impl RuntimeCommandBuilder {
         }
     }
 
-    pub fn add_volume<H, C>(&mut self, host_path: H, container_path: C)
+    pub fn add_volume<H, C>(&mut self, host_path: H, container_path: C, readonly: bool)
```

Comment:
Do we need the functions in this file to be `pub`? Reason I'm not a fan of `pub` all the things is that it doesn't warn you if they're not being used.

Reply:
is `pub(super)` fine?